### ISSUE TITLE
Incompatible with keepass 2.42.1...

### ISSUE THREAD

**Issue by marinerdushyant** [mitchcapper/KPEntryTemplates#63]
With keepass 2.42.1... its not accepting the plugin... not detecting at all... not showing in the plugin section though folder is attached to plugin folder rest all plugin are detecting....

**Comment by mitchcapper** [comment]
You are likely using an old version.  Make sure you remove the old version completely before putting the new one from the releases folder in place.  If that doesn't work try a fresh keepass portable install in a temporary directory with a new download of the KPEntryTemplates from the release section to confirm you are on the latest.

**Comment by marinerdushyant** [comment]
Hi mitch,

Are u talking about older version of KPEntryTempelate?? I download directly
from the plugin page through git, n it is suppose to be the latest version.
Kindly Send me the link across of new version of KPEntryTemplates if any.

I tried to uninstall Keepass and install back full version 2.42.1 and put
the the KPEntryTemplates folder in plug in folder but it's not at all
detecting the plugin neither its saying its not compatible.. Even I tried
with portable one but it's of no success. Kindly suggest or need to wait
for updated version of plugin.

With Warm Regards,
DUSHYANT G. KAMALIA
Add: D-604, Samanvay Residency,
Gala Gymkhana Road,
Opp. Safal Parishar, South Bopal,
Ahmedabad - 380 058. (GUJARAT) -INDIA.
TEL. - +91-2717-403623
MOB. - 09426412661
